src/components/MarkdownProvider/components/Component.tsx

```diff
@@ -7,7 +7,7 @@
 
 import React from 'react';
 import { MD, Span } from '@zendeskgarden/react-typography';
-import { PALETTE } from '@zendeskgarden/react-theming';
+import { PALETTE_V8 } from '@zendeskgarden/react-theming';
```

Comment:
Is the use of `PALETTE_V8` intentional here? I thought we would stick to the new PALETTE and map-to-closest strategy.

Reply:
Yes, please see updated PR description.